### DISCUSSION TITLE
my page is not showing up for some reason

### DISCUSSION THREAD

**Discussion by Amermob** [community/community#34871] [Other Feature Feedback, Questions, & Ideas]
hi, am having problems with my page not showing up for some reason, all am getting is the name of the repository and the title of the html file 

like: https://amermob.github.io/Sombrero-Coffee-Shop/

**Comment by SilasRodrigues19** [comment]
@Amermob Ur html file should preferably be called index.html, otherwise u have to inform the name after the end of the URL generated by GitHub Pages.

If u want to access with the same URL u tried, just change the name of ur html file to ```index.html```, otherwise enter the file name as per this link -> [https://amermob.github.io/Sombrero-Coffee-Shop/Sombrero.html](https://amermob.github.io/Sombrero-Coffee-Shop/Sombrero.html).

**Reply by SilasRodrigues19** [reply]
This is because the default landing page of many web servers defaults the name to ```index.html``` or ```default.htm```
